exchange-wrapper/contracts/ExchangeWrapper.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "@rarible/exchange-v2/contracts/lib/LibTransfer.sol";
+import "@rarible/exchange-v2/contracts/lib/BpLibrary.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@rarible/exchange-interfaces/contracts/IWyvernExchange.sol";
+import "@rarible/exchange-interfaces/contracts/IExchangeV2.sol";
+import "@rarible/royalties/contracts/LibPart.sol";
+
+
+contract ExchangeWrapper is OwnableUpgradeable {
+    using LibTransfer for address;
+    using BpLibrary for uint;
+    using SafeMathUpgradeable for uint;
+
+
+    IWyvernExchange public wyvernExchange;
+    IExchangeV2 public exchangeV2;
+
+    enum Markets {
+        ExchangeV2,
+        WyvernExchange
+    }
+
+    struct TradeDetails {
+        Markets marketId; //if 1 - market is IWyvernExchange, 0 - market is IExchangeV2
```

Comment:
comment is not needed after you introduced enum

Reply:
done